datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/timeline/ListSchemaBlameResolver.java

```diff
@@ -0,0 +1,65 @@
+package com.linkedin.datahub.graphql.resolvers.timeline;
+
+import com.linkedin.common.urn.Urn;
+import com.linkedin.datahub.graphql.generated.ListSchemaBlameInput;
+import com.linkedin.datahub.graphql.generated.ListSchemaBlameResult;
+import com.linkedin.datahub.graphql.types.timeline.mappers.SchemaBlameMapper;
+import com.linkedin.metadata.timeline.TimelineService;
+import com.linkedin.metadata.timeline.data.ChangeCategory;
```

Comment:
Heads up - for new Resolvers we usually try to add unit tests

Reply:
Can this be done as a fast-follow? I have Cypress end-to-end integration tests written that also tests the resolver output

Reply:
yup

Reply:
Awesome